src/dmd/tokens.d

```diff
@@ -910,3 +910,32 @@ extern (C++) struct Token
         return tochars[value];
     }
 }
+
+/** Negate a relational operator, eg >= becomes <
+ * Params:
+ *      op = comparison operator to reverse
+ * Returns:
+ *      reversed operator
+ */
+TOK reverseRelation(TOK op) pure
```

Comment:
```suggestion
auto reverseRelation(TOK op)
```

The benefit that is that the compiler will infer the function attributes. What do you think?